paper-server/patches/sources/net/minecraft/world/inventory/ChestMenu.java.patch

```diff
@@ -26,14 +26,21 @@
 +        this.bukkitEntity = new org.bukkit.craftbukkit.inventory.CraftInventoryView(this.player.player.getBukkitEntity(), inventory, this);
 +        return this.bukkitEntity;
 +    }
++
++    @Override
++    public void startOpen() {
++        this.container.startOpen(this.player.player);
++    }
 +    // CraftBukkit end
  
      private ChestMenu(MenuType<?> type, int containerId, Inventory playerInventory, int rows) {
          this(type, containerId, playerInventory, new SimpleContainer(9 * rows), rows);
-@@ -52,6 +_,9 @@
+@@ -51,7 +_,10 @@
+         checkContainerSize(container, rows * 9);
          this.container = container;
          this.containerRows = rows;
-         container.startOpen(playerInventory.player);
+-        container.startOpen(playerInventory.player);
++        // container.startOpen(playerInventory.player); // CraftBukkit - don't startOpen until menu actually opens
```

Comment:
why // CraftBukkit instead of // Paper ?

Reply:
These changes are from upstream if there is a general consensus I should switch the comments to Paper I will do so.

Reply:
Please do yes :+1: